### PR TITLE
Weeping Stars

### PR DESCRIPTION
**Pull request by Caelphon**
Added the region weeping stars. Needs a stargazer, and skybox.

### CODE REVIEW

**Comment by Caelphon** [comment]
!review

**Comment by TheDanseMacabre** [comment]
The FSF shouldn't be active in this sector, if you ask me

File: maps/away/ships/sol_merc/fsf_patrol_ship.dm [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	name = "FSF Corvette"
 	description = "A small corvette manufactured for the Solarian Navy by Hephaestus, the Montevideo-class is an anti-piracy vessel through and through - with a shuttle bay that takes up a third of the ship and only a single weapon hardpoint located in one arm of the ship, the Montevideo is designed for long-term, self-sufficient operations in inhabited space against small-time pirate vessels that would be unable to overcome the ship's lackluster armaments. Generous automation and streamlined equipment allows it to function with a very small crew."
 	suffixes = list("ships/sol_merc/fsf_patrol_ship.dmm")
-	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE)
+	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE, SECTOR_WEEPING_STARS)
```
Comment (NonQueueingMatt) [inline maps/away/ships/sol_merc/fsf_patrol_ship.dm:5]:
```suggestion
	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE)
```

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Comment by Alberyk** [comment]
Added in another pr.

**Pull request closed without merging**